packages/langium/src/grammar/generated/grammar.ts

```diff
@@ -14,8 +14,8 @@ export const LangiumGrammarGrammar = (): Grammar => loadedLangiumGrammarGrammar
   "rules": [
     {
       "$type": "ParserRule",
-      "name": "Grammar",
       "entry": true,
+      "name": "Grammar",
```

Comment:
Well, they are a side effect. Since we do assignments in different orders now (due to the way fragment rules worked previously), some property keys might swap places. We should maybe ensure a consistent order in the JSON serializer.